src/common/xo/index.js

```diff
@@ -972,12 +972,19 @@ export const addAcl = ({subject, object, action}) => (
   call('acl.add', resolveIds({subject, object, action}))::tap(
     subscribeAcls.forceRefresh
   )
-)
+).catch(err => error('Add ACL', err.message || String(err)))
```

Comment:
I'm not sure for the name, but just to write it lighter :
```
promise::rethrow(err => console.log(err))
```
instead of :
```
promise.catch(error => {
  console.log(error)
  throw error
})
```